dev/services/alchemist/ga_s2_wo_3/s2_wo.alchemist.yaml

```
@@ -0,0 +1,59 @@
+specification:
+  products:
+    - s2a_ard_granule
+    - s2b_ard_granule
+  measurements: ['nbart_blue', 'nbart_green', 'nbart_red', 'nbart_nir_1', 'nbart_swir_2', 'nbart_swir_3', 'fmask']
+  measurement_renames:
+    nbart_nir_1: nbart_nir
+    nbart_swir_2: nbart_swir_1
+    nbart_swir_3: nbart_swir_2
+  resampling:
+    fmask: nearest
+    '*': bilinear
+
+  aws_unsigned: False
+  transform: wofs.virtualproduct.WOfSClassifier
+  transform_url: 'https://github.com/GeoscienceAustralia/wofs/'
+  override_product_family: ard
+  basis: nbart_green
+
+  transform_args:
+    dsm_path:  's3://dea-non-public-data/dsm/dsm1sv1_0_Clean.tiff'
+
+output:
+  location: s3://dea-public-data-dev/derivative/
+  nodata: 1
+  dtype: uint8
+  explorer_url: https://explorer.dev.dea.ga.gov.au
+  preview_image_singleband:
+    measurement: water
+    lookup_table:
+      0: [150, 150, 110]   # dry
+      1: [0, 0, 0]   # nodata,
+      16: [119, 104, 87]   # terrain
+      32: [89, 88, 86]     # cloud_shadow
+      64: [216, 215, 214]  # cloud
+      80: [242, 220, 180]  # cloudy terrain
+      128: [79, 129, 189]  # water
+      160: [51, 82, 119]   # shady water
+      192: [186, 211, 242] # cloudy water
+  write_data_settings:
+    overview_resampling: mode
+  reference_source_dataset: True
+  write_stac: True
+  inherit_geometry: True
+
+  metadata:
+    product_family: wo
+    producer: ga.gov.au
+    dataset_version: 0.0.1
+    collection_number: 3
+    naming_conventions: dea_s2_derivative
+
+  properties:
+    dea:dataset_maturity: interim
+
+processing:
+  dask_chunks:
+    x: 600
+    y: 600
```

We should change this.

```suggestion
    x: 6144
    y: -1
```

Since the shape is `"shape": [10980, 10980]`, and blocksize is 1024, let's do it in halves, like we do the Landsat data. See: https://github.com/opendatacube/datacube-alchemist/blob/main/examples/c3_config_wo.yaml